themes/default/content/learn/abstraction-encapsulation/component-resources/index.md

```diff
@@ -81,14 +154,28 @@ class OurBucketComponent(pulumi.ComponentResource):
         )
         return bucket_policy
 
+bucket1 = OurBucketClass('laura-bucket-1', 'default')
+bucket1.set_policy()
 
-
+pulumi.export("bucket_name", bucket1.bucket.id)
```

Comment:
@nimbinatus and I discovered these lines are missing from what's in production now. 😬 Added here and in one other place.

Reply:
:shame: sorry! Copy-paste error on my part.